rhcos-layer/README.md

```diff
@@ -0,0 +1,88 @@
+# Creating RHCOS image Layer
+
+This folder contains artifacts to create RHCOS image layer for SNP and TDX.
+
+The RHCOS image layer contains the required kernel version, kata and qemu rpms.
+
+## Download the required rpms
+
+For SNP
+
+```sh
+cd rpms-download
+./download-rpms.sh rhel-rpms.yaml
+```
+
+Copy the downloaded rpms to the `snp/rpms` folder
+
+For TDX
+
+```sh
+cd rpms-download
+./download-rpms.sh centos-rpms.yaml
+```
+
+Copy the downloaded rpms to the `tdx/rpms` folder
+
+## Build the RHCOS layer
+
+### With access to OCP cluster
+
+Build for SNP
+
+```sh
+sudo make TEE=snp build
+```
+
+To add the CA certs file (say ca-certs) and Registry auth file (say config.json) to the initrd,
+run the following.
+
+```sh
+sudo make TEE=snp CA_CERTS=ca-certs REG_AUTH=config.json build
+```
+
+Build for TDX
+
+```sh
+make TEE=tdx build
```

Comment:
sudo is needed here otherwise creation of initrd will complain while untarring pause_bundle.